### DISCUSSION TITLE
Unable to set SQS queue in Django project

### DISCUSSION THREAD

**Discussion by dchoruzy** [celery/celery#7407] [General]
Hey,
I'm trying to use Celery in Django application with SQS as a broker.
This is what I added to settings.py file:

```
CELERY_BROKER_URL = "sqs://"
CELERY_BROKER_TRANSPORT_OPTIONS = {
    "region": "eu-central-1",
}
CELERY_DEFAULT_QUEUE = "django_devops_sqs"
```

Problem is that it won't utilize existing "django_devops_sqs" queue but instead it creates new one called "celery"

[a comment by an automated account is not shown]

**Comment by auvipy** [comment]
please follow proper instruction for reporting issues.
